### PR TITLE
DOC-3925 Update Google Tag Manager body script

### PR DESCRIPTION
**Pull request by jgillenwater**

### CODE REVIEW

**Review by bdunn313 — APPROVED** [review]
Looks correct to me! Is everyone aware that an existing GTM tag was in use on the docs site?

File: src/partials/main.hbs [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 {{> toolbar}}
   <div class="content">
-  <!-- Google Tag Manager (noscript) -->
-  <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-PSHTV2W"
-  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
-  <!-- End Google Tag Manager (noscript) -->
+<!-- Google Tag Manager (noscript) -->
+<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-5FSG7Q"
+height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+<!-- End Google Tag Manager (noscript) -->
```
Comment (jgillenwater) [inline src/partials/main.hbs:7]:
```suggestion
  <!-- Google Tag Manager (noscript) -->
  <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-5FSG7Q"
  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
  <!-- End Google Tag Manager (noscript) -->
```

File: src/partials/head-scripts.hbs [hunk 2]
```diff
@@ -7,9 +7,8 @@
     new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
     j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
     'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-    })(window,document,'script','dataLayer','GTM-PSHTV2W');</script>
+    })(window,document,'script','dataLayer','GTM-5FSG7Q');</script>
     <!-- End Google Tag Manager -->
-
```
Comment (eric-schneider) [inline src/partials/head-scripts.hbs:12]: I don't know whether removing this blank line affects anything. It's probably fine?

**Review by eric-schneider — APPROVED** [review]
lgtm, though I did leave a comment.

Note, while I don't think this method of integration will cause us any near-term issues, it appears that we may not be following best practices. I created a ticket to follow up on re-evaluating our Google Tag Manager / Analytics integration: https://datastax.jira.com/browse/DOC-3930

File: src/partials/head-scripts.hbs [hunk 3]
```diff
@@ -7,9 +7,8 @@
     new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
     j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
     'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-    })(window,document,'script','dataLayer','GTM-PSHTV2W');</script>
+    })(window,document,'script','dataLayer','GTM-5FSG7Q');</script>
     <!-- End Google Tag Manager -->
```
Comment (jgillenwater) [inline src/partials/head-scripts.hbs:11]:
```suggestion
    })(window,document,'script','dataLayer','GTM-5FSG7Q');</script>
    <!-- End Google Tag Manager -->
    
```

**Pull request merged**